resdk/tests/unit/test_resolwequerry.py

```diff
@@ -4,19 +4,25 @@
 # pylint: disable=missing-docstring, protected-access
 
 import unittest
+
 import six
+import slumber
 
 from mock import patch, MagicMock
 
 from resdk.resolwe import ResolweQuerry
 
-if six.PY2:
-    # pylint: disable=deprecated-method
-    unittest.TestCase.assertRegex = unittest.TestCase.assertRegexpMatches
-
 
 class TestResolweQuerry(unittest.TestCase):
 
+    @patch('resdk.resolwe.getattr')
+    @patch('resdk.resolwe.Resolwe')
+    @patch('resdk.resources.sample.Sample')
```

Comment:
Yes, unit test should mock everything that comes from outside. It is good approach to write few e2e tests that cover common use cases and then unit tests for every possibility. But I agree that sometimes it is hard (or doesn't make any sense) to mock everything. So use common sense to decide what to mock and what not.